port.yaml

```diff
@@ -5,19 +5,7 @@
 steward: team-devx
 # Any tags relevant to your services and/or mandated by platform teams, such as data classification tags
 tags:
-  # Which type of service the repository defines. 
-  #
-  # Use:
-  #  - `service` for web services (moons)
-  #  - `application` for downloadable software (such as an iOS/Android app,
-  #     CLI tool, standalone executable). This also includes e.g. web apps.
-  #  - `library` for software used by other components
-  #  - `configuration` for repos that contain pure configuration (such as
-  #    setup of DangerJS or Renovate).
-  #  - `documentation` for repos that mostly contain documentation.
-  #  - `hiring-challenge` for our hiring challenges.
-  - key: type
-    value: documentation
+  type: documentation
```

Comment:
**question:** Why are we removing the comment? How will engineers know which are the possible values available?

Reply:
The migration script has two "paths". The repo either has the port.yaml, or it doesn't. If it does, it just fixes some values. If it doesn't have anything, it starts from a template with comments. 
We can add the comment in the template, but if port.yaml is already there, do you think it makes sense to keep the comment?

Reply:
@Niceplace Do we have a task for the general migration that we can add this point about the change in the migration script?